_test/compile-ksy

```diff
@@ -1,3 +1,9 @@
-#!/bin/sh
+#!/bin/sh -e
 
-ksc -t all --outdir=compiled ksy/*.ksy
+ksc \
+	--verbose=file \
+	-t javascript \
+	-t python\
+	-t ruby \
+	--outdir=compiled \
+	ksy/*.ksy
```

Comment:
My personal preference would be to always use space as a parameter name/value separator, as `ksc --help` suggests:

```console
$ ksc --help
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding="UTF-8"
kaitai-struct-compiler 0.10-SNAPSHOT
Usage: kaitai-struct-compiler [options] <file>...

  <file>...                source files (.ksy)
  -t, --target <language>  target languages (graphviz, csharp, rust, all, perl, java, go, cpp_stl, php, lua, python, nim, html, ruby, construct, javascript)
  -d, --outdir <directory>
                           output directory (filenames will be auto-generated)
  (...)
  --verbose <value>        verbose output
```

Note that it's written as `--verbose <value>`, **not ~~`--verbose=<value>`~~**.

```suggestion
ksc \
	--verbose file \
	-t javascript \
	-t python\
	-t ruby \
	--outdir compiled \
	ksy/*.ksy
```

Sorry for nitpicking.